components/iam/base/external-secrets/konflux-ldap-sa.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: konflux-ldap-sa
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+    argocd.argoproj.io/sync-wave: "-1"
+spec:
+  dataFrom:
+    - extract:
+        conversionStrategy: Default
+        decodingStrategy: None
+        key: staging/group-sync/konflux-ldap-sa
```

Comment:
we need to patch this one for the 2 production clusters, should be "production/group-sync/konflux-ldap-sa" for them